deepseg.cc

```diff
@@ -190,6 +204,33 @@ void *grab_thread(void *arg) {
 	return NULL;
 }
 
+ModelType get_modeltype(const char* modelname) {
+	if (strstr(modelname, "body-pix")) {
+		return ModelType::BodyPix;
+	}
+	else if (strstr(modelname, "deeplab")) {
+		return ModelType::DeepLab;
+	}
+	else if (strstr(modelname, "segm_")) {
+		return ModelType::GoogleMeetSegmentation;
+	}
+	else if (strstr(modelname, "selfie")) {
+		return ModelType::MLKitSelfie;
+	}
+	fprintf(stderr, "Unknown model type '%s'.\n", modelname);
+	exit(1);
```

Comment:
Preferably have an `unknown` model type normalized `0..1` and don't exit wildly from within functions. Normally a caller wouldn't expect a function call like `getSomething` to terminate your program.

Reply:
Good point, fixed in last revision.